app/javascript/components/CurriculaSearch.js

```diff
@@ -65,10 +76,15 @@ class CurriculaSearch extends React.Component {
                 data.forEach((standard) => {
                     this.state.checkboxbools.push({
                         id: standard.id,
-                        description: standard.description,
+                        description: standard.id + ": " + standard.description,
                         checked: false
                     })
                 })
+                this.state.checkboxbools.push({
+                    id: "selectall",
+                    description: "Select all of the above",
+                    checked: false
+                })
             }
```

Comment:
can you explain what's going on here? seems like there's a cleaner way to handle this

Reply:
The ID is used to identify which checkbox was selected and then update the corresponding boolean. The description is the entire label for the checkbox. For the standards, we want the label to show the ID and the description. However, the select all box does not follow this format, so only the standards use their ID and description as a label.

Reply:
hmmm i see -- i think we should refactor this because it's still a bit hard to grok. we can come back to this separately but i think we should
not manipulate `this.state` directly. (this can cause unexpected behavior with state transitions -- see here https://stackoverflow.com/questions/35867038/what-the-difference-of-this-state-and-this-setstate-in-reactjs)

Reply:
So leave it for now?